nextflow.config

```diff
@@ -62,11 +62,17 @@ params {
     build_diamond            = false
     build_kaiju              = false
     build_malt               = false
-    malt_sequencetype        = "DNA"
```

Comment:
If we don't use this anymore, we should remove this from the `nextflow_schema.json` file, and references to it elsehwere, (e.g. test profiles etc)

Currently we get the following warning:

```
WARN: Access to undefined parameter `malt_sequencetype` -- Initialise it to a default value eg. `params.malt_sequencetype = some_value`
```